packages/cli-kit/src/node/ruby.ts

```diff
@@ -33,10 +31,9 @@ export async function execCLI2(args: string[], adminSession?: AdminSession) {
     SHOPIFY_CLI_STORE: adminSession?.storeFqdn,
   }
 
-  spawn('bundle', ['exec', 'shopify'].concat(args), {
+  await system.exec('bundle', ['exec', 'shopify'].concat(args), {
```

Comment:
Have you encountered any issues with `system.exec`? I did a similar change but reverted it because @isaacroldan ran into issues when using `system.exec`.

Reply:
I didn't have any such issues, @pepicrft